be/src/formats/orc/fill_function.cpp

```diff
@@ -702,6 +702,75 @@ static void fill_string_column_with_null(orc::ColumnVectorBatch* cvb, ColumnPtr&
     c->update_has_null();
 }
 
+static void fill_varbinary_column(orc::ColumnVectorBatch* cvb, ColumnPtr& col, size_t from, size_t size,
+                                  const TypeDescriptor& type_desc, const OrcMappingPtr& mapping, void* ctx) {
+    auto* data = down_cast<orc::StringVectorBatch*>(cvb);
+
+    size_t len = 0;
+    for (size_t i = 0; i < size; ++i) {
+        len += data->length[from + i];
+    }
+
+    int col_start = col->size();
+    auto* values = ColumnHelper::cast_to_raw<TYPE_VARBINARY>(col);
+
+    values->get_offset().reserve(col_start + size);
+    values->get_bytes().reserve(values->get_bytes().size() + len);
+
+    auto& vb = values->get_bytes();
+    auto& vo = values->get_offset();
+    int pos = from;
+
+    for (int i = col_start; i < col_start + size; ++i, ++pos) {
+        vb.insert(vb.end(), data->data[pos], data->data[pos] + data->length[pos]);
+        vo.emplace_back(vb.size());
+    }
+}
+
+static void fill_varbinary_column_with_null(orc::ColumnVectorBatch* cvb, ColumnPtr& col, size_t from, size_t size,
+                                            const TypeDescriptor& type_desc, const OrcMappingPtr& mapping, void* ctx) {
+    auto* data = down_cast<orc::StringVectorBatch*>(cvb);
```

Comment:
only support String?

Reply:
orc string and binary use the same type of `ColumnVectorBatch`

https://github.com/letian-jiang/starrocks/blob/0617986c708fcb06cafdcfee1a27089c189b1640/be/src/formats/orc/apache-orc/c++/src/TypeImpl.cc#L327
https://github.com/StarRocks/starrocks/blob/0617986c708fcb06cafdcfee1a27089c189b1640/be/src/formats/orc/apache-orc/c++/src/TypeImpl.cc#L327